src/components/YouAndYourFamilyCard.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import ParticipantInfo from './ParticipantInfo';
+import participantData from './data/participantData.js';
+import '../index.css';
+import OverviewCard from './OverviewCard';
+
+/**
+ * Issues
+ * - cannot get 'add' btn to show inside the OverviewCard
+ * - cannot get 'add' to open the popup (somethings wrong with communication between children and parents)
+ */
+
+function YouAndYourFamilyCard(props) {
+    const participantComponents = participantData.map(data => <ParticipantInfo key={data.id} name={data.name} age={data.age} dutypreferences={data.dutypreferences} carseat={data.carseat}/>);
+    const CardHeader = ["Name","Age","Duties","Carseat"];
+    const Rightbuttons = ["Add"];
+    const LeftButtons = ["Delete"];
+
+    return (
+        <div className="YouAndYourFamily">
```

Comment:
CSS class names are conventionally named with kebab case:
you-and-your-family

Only JavaScript classes and React Component uses CamelCase
Variables and functions are lowerCamelCase